systems.csv

```diff
@@ -583,6 +603,7 @@ FR,Citiz Nantes,Nantes,citiz_nantes,https://citiz.coop/,https://backend.citiz.fr
 FR,Citiz Occitanie,Toulouse,citiz_occitanie,https://citiz.coop/,https://backend.citiz.fr/public/provider/8/gbfs/v3.0/gbfs.json,3.0,,,
 FR,Citiz Provence,Marseille,citiz_provence,https://citiz.coop/,https://backend.citiz.fr/public/provider/6/gbfs/v3.0/gbfs.json,3.0,,,
 FR,Citiz Rennes Métropole,Rennes,citiz_rennes_metropole,https://citiz.coop/,https://backend.citiz.fr/public/provider/19/gbfs/v3.0/gbfs.json,3.0,,,
+FR,Compte Mobilité,Mulhouse,mulhouse,https://www.compte-mobilite.fr/,https://api.cyclocity.fr/contracts/mulhouse/gbfs/v3/gbfs.json,2.3; 3.0,,,

```

Comment:
VeloCité Mulhouse has been taken over by Nextbike. This old JCDecaux feed has 0 available vehicles.